### DISCUSSION TITLE
Requests to adguard.com

### DISCUSSION THREAD

**Discussion by ali27868** [AdguardTeam/AdGuardHome#3919] [Q&A] [answered]
Hi!

I have AGH running on the router and Adguard on PC (DNS option off).

One of the top queried domains I see in AGH are local.adguard.com and api.adguard.com. What are these requests - can you please explain what goes on behind the scenes?

Thank you.

**Comment by ainar-g** [comment, marked as answer]
These are probably requests from the AdGuard program on your PC, since AdGuard only checks `static.adguard.com` for software updates.
